### ISSUE TITLE
[Issue] Crash on Startup / CodeChicken Issue

### ISSUE THREAD

**Issue by accessdenied0** [Samuel-Harbord/SpectriteMod#5]
After I added Spectrite to my modlist,[ I couldn't load up my instance anymore.](https://pastebin.com/rVA796yV)

MC version: 1.12
Mod version 1.3.2
Forge: 14.21.1.2407

access_denied

**Comment by Flashfyre** [comment]
Interesting. I was playing around with CCL for a bit, then removed it. Perhaps I accidentally left a remnant of it which could be causing this. I will take a look at this tonight.

**Comment by Flashfyre** [comment]
Could you do me a favour and test this with version 1.2.0? I had not done anything with CCL at that point. If  it works, that will really point me in the right direction.

**Comment by accessdenied0** [comment]
This version is loading and working.

**Comment by Flashfyre** [comment]
Alright, that helps a lot.

**Comment by Flashfyre** [comment]
I'm not seeing anything CCL related anywhere in my mod's directory. I've contacted the maintainer of CCL and, if I can't get anywhere with that, I'll start again with a clean MDK.

**Comment by accessdenied0** [comment]
I might figured out what is gone wrong here and it is not yours mod fault so far I can tell. I run into the same issue after updating some mods. And because my instance runs on nearly max ID's, I installed the NotEnoughID's mod. 

Now I guess what is happening here is: If I use a newer version of your mod, your mod uses the NEID mod's extra IDs but this _somehow_ doesn't like the CCL. 

I'm sure to 80% that what happens here and [I made a post on NEID.](https://github.com/fewizz/NotEnoughIDs/issues/63)
